src/swagger/api.yaml

```diff
@@ -2133,181 +2184,926 @@ paths:
+      operationId: listEmailTemplates
+      description: List email templates in your organization with pagination.
+      security:
+      - api_token: []
+      parameters:
+      - schema:
+          type: string
+        in: query
+        description: Specifies the pagination cursor for the next page of email templates.
+        name: after
+      - schema:
+          type: integer
+          default: 20
+        in: query
+        name: limit
+        description: Specifies the number of results returned (maximum 200)
+      tags:
+      - Brand
+  /api/v1/brands/{brandId}/templates/email/{templateName}:
+    get:
+      summary: Get Email Template
+      responses:
+        '200':
+          description: Successfully fetch an email template by name.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EmailTemplate'
+      operationId: getEmailTemplate
+      description: Fetch an email template by `templateName`
+      security:
+      - api_token: []
+      tags:
+      - Brand
+    parameters:
+    - schema:
+        type: string
+      name: brandId
+      in: path
+      required: true
+    - schema:
+        type: string
+      name: templateName
+      in: path
+      required: true
+  /api/v1/brands/{brandId}/templates/email/{templateName}/default-content:
+    get:
+      summary: Get Default Content of Email Template
+      responses:
+        '200':
+          description: Successfully fetch the default content for an email template.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EmailDefaultContent'
+      operationId: getEmailTemplateDefaultContent
+      description: Fetch the default content for an email template.
+      security:
+      - api_token: []
+      tags:
+      - Brand
+    parameters:
+    - schema:
+        type: string
+      name: brandId
+      in: path
+      required: true
+    - schema:
+        type: string
+      name: templateName
+      in: path
+      required: true
+  /api/v1/brands/{brandId}/templates/email/{templateName}/default-content/preview:
+    parameters:
+    - schema:
+        type: string
+      name: brandId
+      in: path
+      required: true
+    - schema:
+        type: string
+      name: templateName
+      in: path
+      required: true
+    get:
+      summary: Get Preview of Email Template Default Content
+      responses:
+        '200':
+          description: ' Successfully fetch the preview of an email template''s default content.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EmailPreview'
+      operationId: previewEmailDefaultContent
+      security:
+      - api_token: []
+      description: Fetch a preview of an email template's default content by populating velocity references with the current user's environment.
+      tags:
+      - Brand
```

Comment:
These 2 operations work incorrectly. 
```
{
    "errorCode": "E0000022",
    "errorSummary": "The endpoint does not support the provided HTTP method",
    "errorLink": "E0000022",
    "errorId": "oae7Vnow73hRsS1TYMd0VVxPw",
    "errorCauses": []
}
```

Reply:
I need to add lifecycle flags, these APIs did not exist at the time.